binding/json.go

```diff
@@ -32,7 +37,7 @@ func (jsonBinding) Name() string {
 
 func (jsonBinding) Bind(req *http.Request, obj interface{}) error {
 	if req == nil || req.Body == nil {
-		return errors.New("invalid request")
+		return InvalidRequest
```

Comment:
How about:
```suggestion
		return ErrInvalidRequest
```
I think it's a common way to name errors.

Reply:
ok,thinks @kszafran